Sources/Testing/Expectations/Expectation+Macro.swift

```diff
@@ -69,6 +69,33 @@
   sourceLocation: SourceLocation = SourceLocation()
 ) -> T = #externalMacro(module: "TestingMacros", type: "RequireMacro")
 
+/// Unwrap an optional boolean value or, if it is `nil`, fail and throw an
+/// error.
+///
+/// - Parameters:
+///   - optionalValue: The optional value to be unwrapped.
+///   - comment: A comment describing the expectation.
+///   - sourceLocation: The source location to which recorded expectations and
+///     issues should be attributed.
+///
+/// - Returns: The unwrapped value of `value`.
```

Comment:
```suggestion
/// - Returns: The unwrapped value of `optionalValue`.
```

And a few other references to `value` in surrounding lines

Reply:
Resolved.